#Print numbers fron 1 to 20 using a while loop.

In [2]:
num = 1
while num <= 20:
    print(num)
    num += 1

1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20


#Print every 3rd bumber from 3 to 30.

In [3]:
for num in range(3, 31, 3):
    print(num)

3
6
9
12
15
18
21
24
27
30


#Print the sum of even numbers from 2 to 20.

In [7]:
sum_even = 0
for num in range(2, 21, 2):
    sum_even += num
print(sum_even)

110


#Print the product of odd numbers from 1 to 9.

In [8]:
product = 1
for num in range(1, 10, 2):
    product *= num
print(product)

945


#Create a list of 5 names and print each on a new line.

In [9]:
names = ["Seerat", "Farwa", "Meerab", "Faiza", "Ayesha"]
for name in names:
    print(name)

Seerat
Farwa
Meerab
Faiza
Ayesha


#Count how many letters 'a' are in the world "banana".

In [11]:
word = "pineapple"
count = word.count('a')
print("Count of 'a':", count)

Count of 'a': 1


#Print only the negative numbers from a given list.

In [12]:
numbers = [3, -5, -7, 1, 8, -9]
for num in numbers:
    if num < 0:
        print(num)

-5
-7
-9


#

#Print only the numbers ending in 5 from 1 to 50.

In [16]:
for num in range(1, 51):
    if num % 10 == 5:
        print(num)

5
15
25
35
45


#Print the length of each word in the list["cat", "elephant", "bat"].

In [18]:
words = ["cat", "elephant", "bat"]
for word in words:
    print(len(word))

3
8
3


# Take 3 numbers as input from the user and print their average.

In [19]:
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))
num3 = int(input("Enter third number: "))
average = (num1 + num2 + num3) / 3
print(average)

43.666666666666664


#Print the table of 5 in reverse (from 5x10 to 5x1).

In [23]:
for i in range(10 , 0, -1):
    print(f"5 x {i} = {5 * i}")

5 x 10 = 50
5 x 9 = 45
5 x 8 = 40
5 x 7 = 35
5 x 6 = 30
5 x 5 = 25
5 x 4 = 20
5 x 3 = 15
5 x 2 = 10
5 x 1 = 5


#Use a for loop to print all characters in "Python Is fun".

In [39]:
text = "Python Is fun"
for char in text:
    print(char)

P
y
t
h
o
n
 
I
s
 
f
u
n


#Create a list of numbers and print every second item.

In [26]:
numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
print("Every second item: ")
print(numbers[1::2])

Every second item: 
[2, 4, 6, 8, 10]


#Take a word from the user and count how many consonants it has.

In [29]:
word = input("Enter a word: ")
consonants = "abcdefghigklmnoprstwyxz"
count = 0
for char in word:
    if char in consonants:
        count += 1
print(f"Number of consonants: {count}")

Number of consonants: 5


#Print the cubes of numbers from 1 to 5.

In [31]:
for i in range(1, 6):
    print(f"{i} is {i*3}")

1 is 3
2 is 6
3 is 9
4 is 12
5 is 15


#Take a list of words and print those that start with 'a'.

In [40]:
words = ["apple", "cherry", "apricot", "avocado"]
print("Words starting with 'a':")
for word in words:
    if word.startswith("a"):
        print(word)

Words starting with 'a':
apple
apricot
avocado


#Use a loop to print numbers between 100 and 110 (excluding 105).

In [34]:
for i in range(100, 111):
    if i == 105:
        continue
    print(i)

100
101
102
103
104
106
107
108
109
110


#Print the first 5 positive multiples of 4.

In [35]:
for i in range(1, 6):
    print(4 * i)

4
8
12
16
20


#Print each word in the sentence "acoding is easy" on a new line.

In [36]:
sentence = "Coding is easy"
words = sentence.split()
for word in words:
    print(word)

Coding
is
easy


#**Create a number list and print only even numbers using conditions in loop.

In [37]:
numbers = [1, 2, 4, 3, 5, 7, 6, 8, 9, 10]
print("Even numbers: ")
for num in numbers:
    if num % 2 == 0:
        print(num)

Even numbers: 
2
4
6
8
10
